### ISSUE TITLE
Include a resource for tracking fueling stops

### ISSUE THREAD

**Issue by matthesrieke** [enviroCar/enviroCar-server#175]
This is to support Julius with his Bachelor thesis. He implemented a "Fueling stops logbook". Its data should be integrated (storage, accessibility) into the server.

Target resource: https://envirocar.org/api/dev/users/theuser/fuelings

POST:

``` JSON
{
   "Cost:":"60.88",
   "Fueltype:":"diesel",
   "VolumeUnit:":"Liter",
   "Volume:":"44.8",
   "Name:":"AlbertRemke",
   "DistanceUnit:":"Km",
   "Mileage:":"26809",
   "Currency:":"Euro",
   "Car:":"51eff086e4b058cd3d5b2303",
   "DateTime:":"31-01-2014 19:27",
   "MissedFuelStop": false,
   "Comment":"Some generic string comment"
}
```

Note: the format is his draft and can be discussed. E.g. it is not required to have it flat. Another element for geolocation might also be added.

Mandatory elements:
- Cost (+Unit)
- Fueltype
- Volume (+Unit)
- Name (well this is redundant, isn't it?)
- Mileage (+Unit)
- Car
- DateTime

Optional:
- MissedFuelStop
- Comment

GET should return it basically the same way.

Rights management: Only the user can POST and GET on the resource. This is a complete private resource.

If required, I can provide some semantics on the fields.

**Comment by autermann** [comment]
Done in #176
